### ISSUE TITLE
Folders!!!

### ISSUE THREAD

**Issue by wiktorelka** [TCNOco/TcNo-Acc-Switcher#230]
Please consider making **folders** for TcNo Account Switcher, it's really annoying when i have over 30 steam accounts and i need to look for the one i want to play on, folders would really help me organize things.

**Comment by klNuno** [comment]
Sorry to bump this thread but I completely agree with this user, this would be a big QoL improvement

**Comment by TCNOco** [comment]
> @TCNOco It works thank you so much ! <333
> 
> Is it possible that you can check this again ? idk if it's complicated to do but that would be really helpful as well #230

Yes. It was a good idea, just never got around to it!

I use https://github.com/lukasoppermann/html5sortable for the sorting, and it works well. I'm not sure if it supports or has functionality for folders. I know there is "nesting" but it's more for lists with an item that has multiple items within. While that's technically what this is, I'd like for a double-click of a folder to open a modal with more accounts in it. It seems simple to imagine, but putting it together takes some willpower that was more than drained out of me by forces outside my control... More below.

More info:

 was working a ton in [JsToRazor-DI-MAUI](https://github.com/TCNOco/TcNo-Acc-Switcher/tree/JsToRazor-DI-MAUI) - a ton of work into an overhaul with QOL improvements. I'm sure I looked and worked on this there... until I hit a brick wall with: [dotnet/maui/issues/10185](https://www.github.com/dotnet/maui/issues/10185), moved to [MicrosoftEdge/WebView2Feedback/issues/2805](https://www.github.com/MicrosoftEdge/WebView2Feedback/issues/2805) then to [microsoft/microsoft-ui-xaml/issues/7366](https://www.github.com/microsoft/microsoft-ui-xaml/issues/7366) where it still hangs in the air if DRAGGING AND DROPPING works or not.... While dragging FILES into and out of a WebView2 window was addressed, I don't know about dragging within....

So these issues lead to a halt in development after putting in 80 commits (24,843 additions and 17,254 deletions) between July 14, 2022 and Sep 20, 2022.

I've worked on it here and there but have infinitely less free time. Now I've got a whole lot done in just a few days.

I managed to bring a lot back from what I put in there but there's still tons I haven't brought across... like

[Modals being draggable](https://github.com/TCNOco/TcNo-Acc-Switcher/commit/22dd6c66ef8483be47413a33c86ac0d96d0548bc) (those popup windows asking for input)
[Switch and Launch](https://github.com/TCNOco/TcNo-Acc-Switcher/commit/31a4cfcee6086721a2e4aed8d32c558a94814814) to switch accounts and launch a game.

tldr: This is why I stopped working so much on the TcNo Account Switcher, besides having less free time. This feature as well as others were worked on (others being added entirely) but stopped by external forces.

Yeah,  I'd like to add this and I'll be getting to it... sometime...

**Comment by klNuno** [comment]
Thank you for letting me know ! I wasn't aware of all of that. It's understandable and I'm glad you still have some free time to fix the acc switcher !

Maybe adding tags to accounts and filter them with the tags could do the job ? It's not as pleasant as having folders but it could be a great alternative.

**Comment by klNuno** [comment]
> Thank you for letting me know @TCNOco ! I wasn't aware of all of that. It's understandable and I'm glad you still have some free time to fix the acc switcher !
> 
> Maybe adding tags to accounts and filter them with the tags could do the job ? It's not as pleasant as having folders but it could be a great alternative.

Hey @TCNOco hope you're doing fine ! Just bumping this up in case you haven't seen it

**Comment by TCNOco** [comment]
Folders ended up not being the best solution from what I can tell - The app underwent a major rewrite and there are tons of new features.
One of them is Tags. You can apply tags to any account, then sort and filter with them. Some are automatically added like cooldowns for CS2, and others can have expiry timers and more.

Should be a lot more useful without adding too much clutter, and still offer folder-like functionality.

<img width="1020" height="643" alt="TcNo-Acc-Switcher_SCXiB1bmOm" src="https://github.com/user-attachments/assets/cae22e82-4196-457f-bcdf-b35afde6e5bc" />

If you're still interested check out the latest pre-releases:
https://github.com/TCNOco/TcNo-Acc-Switcher/releases

**Comment by klNuno** [comment]
hello, thanks I will definitely check it out !

In the meantime I've decided to make [my own thing](https://github.com/klNuno/accshift) just because of the folders xd but it's nice to see that you're still maintaining it :)
